scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -442,22 +442,20 @@ task M2 {
         touch bamout.bam
         touch f1r2.tar.gz
         touch dataset.txt
-        echo "" > normal_name.txt
-
-        gatk --java-options "-Xmx~{command_mem}m" GetSampleName -R ~{ref_fasta} -I ~{tumor_reads} -O tumor_name.txt -encode \
-        ~{"--gcs-project-for-requester-pays " + gcs_project_for_requester_pays}
-        tumor_command_line="-I ~{tumor_reads} -tumor `cat tumor_name.txt`"
 
         if [[ ! -z "~{normal_reads}" ]]; then
-            gatk --java-options "-Xmx~{command_mem}m" GetSampleName -R ~{ref_fasta} -I ~{normal_reads} -O normal_name.txt -encode \
+            gatk --java-options "-Xmx~{command_mem}m" GetSampleName -R ~{ref_fasta} -I ~{normal_reads} -O normal_names.txt -encode \
             ~{"--gcs-project-for-requester-pays " + gcs_project_for_requester_pays}
-            normal_command_line="-I ~{normal_reads} -normal `cat normal_name.txt`"
+            # add "-normal " to the start of each line and " " to the end, then remove newlines
+            # to get -normal sample1 -normal sample2 etc
+            normal_sample_line=`sed 's/^/-normal /' normal_names.txt | sed 's/$/ /' | tr -d "\n"`
```

Comment:
I think this should be equivalent and slightly less cryptic (unless you hate `awk`):
```
awk '{ print "-normal", $0 }' normal_names.txt | tr '\n' ' '
```

Reply:
done.